python/cli/prelude_cli/views/build.py

```diff
@@ -293,9 +309,9 @@ def create_threat_hunt(controller, name, query, test, threat_hunt_id):
 
 @build.command('update-threat-hunt')
 @click.argument('threat_hunt')
-@click.option('-n', '--name', help='Name of this threat hunt query', required=True, type=str)
-@click.option('-q', '--query', help='Threat hunt query', required=True, type=str)
-@click.option('-t', '--test', help='ID of the test this threat hunt query is for', required=True, type=str)
+@click.option('-n', '--name', help='Name of this threat hunt query', type=str)
+@click.option('-q', '--query', help='Threat hunt query', type=str)
+@click.option('-t', '--test', help='ID of the test this threat hunt query is for', type=str)
```

Comment:
are these not required?

Reply:
in an update, users should only set the fields they want to change - if they want to leave the value as is, they dont set it